.env.example

```diff
@@ -10,10 +10,28 @@ TOKEN_KEY=TheTokenKey
 MAILER_API_KEY=theTokenOfTheMailerService
 MAILER_EMAIL=theEmailUsedByTheMailerAsSender
 API_BASE_URL=theApiBaseUrlToInsertInTheLinksSharedInEmails
-TOKEN_KEY=TheTokenKey
```

Comment:
@Ramadhan0 why have you removed this env variable?

Reply:
it is there on line 8